tests/hil/tests/settings/test.c

```diff
@@ -133,6 +151,26 @@ void hil_test_entry(const struct golioth_client_config *config)
             GLTH_LOGI(TAG, "Settings have been reregistered");
         }
 
+        if (golioth_sys_sem_take(restart_sem, 0))
+        {
+            // Delay to ensure RPC response makes it to cloud
+            golioth_sys_msleep(1000);
+
+
+            golioth_client_stop(client);
+
+            golioth_sys_msleep(15 * 1000);
+
+            golioth_client_start(client);
+
+            golioth_sys_msleep(5 * 1000);
+
+            GLTH_LOGI(TAG, "Client restarted");
```

Comment:
```suggestion
            golioth_sys_msleep(1000);

            golioth_client_stop(client);
            golioth_sys_msleep(15 * 1000);

            golioth_client_start(client);
            golioth_sys_msleep(5 * 1000);
            GLTH_LOGI(TAG, "Client restarted");
```

Remove some of the blank lines.